README.md

```diff
@@ -1,6 +1,6 @@
 # Static KAS
 
-A fake kube-apiserver that serves static data from an Openshift must-gather. Dynamically discovers resources and supports logs.
+A fake kube-apiserver that serves static data from an Openshift must-gather. Dynamically discovers resources and supports logs. Requires golang >= 1.17.
```

Comment:
What surprises me is that the go compiler even compiles with an older go, the `go mod` explicitly specifies go 1.17